parser.y

```diff
@@ -10771,4 +10782,70 @@ LoadStatsStmt:
 		}
 	}
 
+/*******************************************************************************************/
+
+IndexAdviseStmt:
+	"INDEX" "ADVISE" LocalOpt "INFILE" stringLit MaxMinutes MaxIndexNum Lines
```

Comment:
Could you please put the `INDEX ADVISE` syntax here?

```
INDEX ADVISE
  [LOCAL]
  INFILE 'file_name'
  [MAX_MINUTES number]
  [MAX_IDXNUM
     [PER_TABLE number] 
     [PER_DB number]
  ]
  [LINES 
    [STARTING BY 'string'] 
    [TERMINATED BY 'string']
  ]
```

BTW, I think `MaxMinutesOpt` and `MaxIndexNumOpt` is better.

Reply:
Changing the name is a good idea!
And...did you mean adding a link to the document? Or write the syntax here directly as a comment in `parser.y`?

Reply:
https://github.com/pingcap/parser/blob/3611952bf747dcbecc9197f85efe37654de5da5b/parser.y#L3779-L3792

Here is an example. 😄